### PR TITLE
website/blogs: blog about sso tax

### PR DESCRIPTION
**Pull request by tanberry**
this week's blog about security taxes

### CODE REVIEW

File: website/blog/2023-10-18-taxed-out-of-security/item.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+Research from Grip, an identity risk management company, shows how the [lack of good options](https://www.grip.security/blog/why-sso-doesnt-protect-80-of-your-saas) plays out. Grip’s research shows that 80% of the SaaS applications employees use are not in their companies’ SSO portals. In their interviews with CISOs, SSO licensing costs – i.e., the SSO tax – were the top reason.
+
+The same logic that applies to the SSO tax also applies to two other security taxes: the MFA tax and the user tax.
+
+Security experts widely agree that SSO is essential for security, but multi-factor authentication is more basic still, making the act of charging extra for MFA even more egregious. But, as we saw in the intro, companies like Box charge extra for multiple authentication methods.
+
+The user tax is more subtle. Companies are charging a user tax when they charge exorbitant amounts more per additional user that customers feel motivated to share the credentials to one account. To be clear, many companies charge on a per-user or per-seat basis (including Authentik), so you can’t call it a tax until the additional costs really become exorbitant.
```
Comment (kensternberg-authentik) [inline website/blog/2023-10-18-taxed-out-of-security/item.md:68]: I don't think I could diagram that second sentence.  It doesn't structure in my head well.

**Review by kensternberg-authentik — APPROVED** [review]
I'm going to approve, but I would recommend fixing that one ungrammatic sentence. (And, _yes, Chrome_, if "ungrammatical" is a word in your dictionary then "ungrammatic" ought to be as well.)

File: website/blog/2023-10-18-taxed-out-of-security/item.md (hunk 1 above)
Comment (tanberry) [reply website/blog/2023-10-18-taxed-out-of-security/item.md:68]: Which sentence exactly do you wnat edited, @kensternberg-authentik ?

Comment (tanberry) [inline website/blog/2023-10-18-taxed-out-of-security/item.md:68]:
```suggestion
The user tax is more subtle. When companies charge excessive amounts per-user to secure those users' accounts, users must either choose to pay the security tax or engage in the highly insecure practice of sharing credentials among several individuals. To be clear, many companies charge on a per-user or per-seat basis (including Authentik), so you can’t call it a tax until the additional costs really become exorbitant.
```

Comment (tanberry) [reply website/blog/2023-10-18-taxed-out-of-security/item.md:68]: Thanks to @kensternberg-authentik for the rewrite here!

**Pull request merged**